hazelcast/src/main/java/com/hazelcast/map/impl/SimpleEntryView.java

```diff
@@ -257,6 +273,7 @@ public void readData(ObjectDataInput in) throws IOException {
         // reads the deprecated evictionCriteriaNumber from the data input (client protocol compatibility)
         in.readLong();
         ttl = in.readLong();
+        maxIdle = in.readLong();
```

Comment:
RU compatibility with 3.10? Seems should check cluster-version like [this](https://github.com/hazelcast/hazelcast/blob/9ae4484b402aa769ec06410a147848c016d2bd7f/hazelcast/src/main/java/com/hazelcast/config/MapConfig.java#L1104)